packages/core/core/src/public/Dependency.js

```diff
@@ -39,6 +41,11 @@ export default class Dependency implements IDependency {
     internalDependencyToDependency.set(dep, this);
   }
 
+  // $FlowFixMe
+  [inspect]() {
+    return `Dependency(${this.sourcePath} -> ${this.filePath})`;
```

Comment:
This is great. Can it include the `moduleSpecifier`?

Maybe we could use these when generating node labels in graphviz 🤔

Reply:
Actually a typo, `this.filePath` doesn't exist and should be `this.moduleSpecifier`.

This string wouldn't be very useful in the graph because `sourcePath` is already implied by the incoming edge, so this would make the labels much longer without providing more information.